contrib/cmake/vcpkg_extra_ports/ffmpeg/4.2-3/0001-create-lib-libraries.patch

```diff
@@ -1,25 +0,0 @@
-diff --git a/configure b/configure
-index 34c2adb..1c8008a 100755
---- a/configure
-+++ b/configure
```

Comment:
github is showing this file and a lot of other vcpkg_extra_port files being removed now, if we merge to develop - but we don't want to break any existing usage of those files.   Can we arrange for no pending changes to those files except where necessary for this PR?   thanks

Reply:
Restored, let me know if I missed any